cmd/client/resources/object.go

```diff
@@ -47,30 +60,39 @@ func ObjectAPIResources() ([]*api.APIResource, error) {
 	if err != nil {
 		return nil, err
 	}
+	globalAPIResources = res
 	return res, nil
 }
 
-func defaultObjectNameSpace() string {
-	return cluster.TrafficNamespace(cluster.NamespaceDefault)
+// trafficObjectStatusNamespace return namespace of traffic object status.
```

Comment:
I can't understand this comment, what does it mean? And the following line also contains confusing grammar.

Reply:
for non-traffic objects, like autocertmanager, it is in namespace default, and its status is in namespace default. 

but for traffic objects, they are managed by traffic controller, and traffic controller use specially prefix for status. for example, if a pipeline in namesapce default, then its status in namespace eg-traffic-default.